bundle/edu.gemini.model.p1/src/main/scala/edu/gemini/model/p1/dtree/Root.scala

```diff
@@ -44,6 +44,6 @@ object Root {
 
   // REL-3769: GPI not available in 2020B.
   // GHOST not yet available
-  val choices: List[Instrument] = List(Alopeke, Flamingos2, /*Ghost,*/ GmosNorth, GmosSouth, Gnirs, Graces, Gsaoi, Igrins, MaroonX, Nifs, Niri, Visitor, Zorro)
+  val choices: List[Instrument] = List(Alopeke, Flamingos2, Ghost, GmosNorth, GmosSouth, Gnirs, Graces, Gsaoi, Igrins, MaroonX, Nifs, Niri, Visitor, Zorro)
```

Comment:
Maybe we need to turn this back off for now?

Reply:
It was uncommented already for PIT 2023B

Reply:
@cquiroz We now want to leave GHOST commented out correct?

Reply:
Yes, it is commented out

Reply:
We are going to want to release the GHOST P2 templates in the June release, supporting GHOST in the _2023A_ PIT. There will also probably be a new 2023B PIT release later in the year with GHOST available. I will be updating the GHOST template rules soon.